search/routes/api/tests/test_api.py

```diff
@@ -22,33 +19,12 @@ class TestAPISearchRequests(TestCase):
 
     def setUp(self):
         """Instantiate and configure an API app."""
-        jwt_secret = "foosecret"
-        os.environ["JWT_SECRET"] = jwt_secret
         self.app = factory.create_api_web_app()
-        self.app.config["JWT_SECRET"] = jwt_secret
         self.client = self.app.test_client()
 
         with open(self.SCHEMA_PATH) as f:
             self.schema = json.load(f)
 
-    @skip("auth scope currently disabled for API")
-    def test_request_without_token(self):
-        """No auth token is provided on the request."""
-        response = self.client.get("/")
-        self.assertEqual(response.status_code, HTTPStatus.UNAUTHORIZED)
-
-    @skip("auth scope currently disabled for API")
-    def test_with_token_lacking_scope(self):
-        """Client auth token lacks required public read scope."""
-        token = helpers.generate_token(
-            "1234",
-            "foo@bar.com",
-            "foouser",
-            scope=[Scope("something", "read")],
-        )
-        response = self.client.get("/", headers={"Authorization": token})
-        self.assertEqual(response.status_code, HTTPStatus.FORBIDDEN)
-
     @mock.patch(f"{factory.__name__}.api.api")
     def test_with_valid_token(self, mock_controller):
```

Comment:
Please change the name of this test if it is no longer testing with a valid token.